lib/types/types.ts

```diff
@@ -56,12 +56,14 @@ export interface FieldDefinition {
     optional?: boolean;
     repeated?: boolean;
     fields?: SchemaDefinition;
+    statistics?: any
 }
 
 export interface ParquetField {
     name: string;
     path: string[];
-    key: string;
+    // key: string;
+    statistics?: any
```

Comment:
These statistics need to be changed from any but I added them since the schema was adding them as one of it's elements.

Reply:
They are used at line 118 and 165

Reply:
The key here is commented out since it's a required element but was conflicting with schema. Maybe put back in as optional depending on use

Reply:
Statistics could possibly be found in https://github.com/LibertyDSNP/parquetjs/blob/chore/schema-conversion/28/gen-nodejs/parquet_types.js